lab5d.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+using namespace std;
+#include "lab5d.h"
+
+
+
+//beginning
```

Comment:
unnecessary annotated code

Reply:
It's note for me, so i don't want to delete it